_api_app/app/Shared/Helpers.php

```diff
@@ -326,4 +326,26 @@ public static function api_response($message='', $data=[], $status=200) {
             'data' => $data
         ], $status);
     }
+
+    /**
+     * Converts array of attributes to string of attributes
+     */
+    public static function arrayToAttributes($attributes)
```

Comment:
Maybe we could call it `arrayToAttributesString` to identify, that we're building a string. Or maybe `arrayToHtmlAttributes`?